source/github_ee/github_ee_test.go

```diff
@@ -0,0 +1,36 @@
+package github_ee
+
+import (
+	"net/http"
+	"net/http/httptest"
+	nurl "net/url"
+	"strings"
+	"testing"
+)
+
+func Test(t *testing.T) {
+	ts := httptest.NewTLSServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		p := strings.Join(strings.Split(r.URL.Path, "/")[:3], "/")
+
+		if p != "/api/v3" {
+			t.Fatal()
+		}
+
+		w.Header().Set("Content-Type", "application/json")
+		w.WriteHeader(http.StatusOK)
```

Comment:
For a more complete mock of the github API, do whatever other API data checks are necessary before returning `http.StatusOK`

Reply:
So I've added a stricter verification to make sure the github api is called with the right path and params. I think that's enough for this test, since `github_ee` doesn't really do anything else.